feRotatorMount.py

```diff
@@ -0,0 +1,239 @@
+"""
+Rotator Mount Control from MIDAS Code
+Skylar Wingfelder 8-23-2021
+This code reads and reports the position of the rotator mount periodically and watches the ODB to see if the 
+Destination changes. If so it moves the mount to the new position.
+"""
+
+import midas
+import midas.frontend
+import midas.event
+
+import collections
+import time
+import serial
+
+#Code which converts a float corresponding to an angle to an 8 character hexidecimal number which indicates how many counts correspond to this angle in the rotation mount software
+def deg2tx (angle):
+    
+    print("d2t angle: ",angle) #print commands for trouble shooting all can be removed
+    degPerCount = 0.0025 # degrees per count from spec sheet
+    counts=int(angle/degPerCount) # integer of the number in counts 
+    print("d2t counts: ", counts)
+    
+    
+    
+    #if the angle is negative the 2s complement must be found before converting the number to hexidecimal
+    if (counts < 0):        
+        counts=counts*-1 #remove negative sign
+        F8=int('FFFFFFFF',16) #Maximum hexidecimal number 
+        twoscomp=F8-counts #subtract counts from max number
+        twoscomp=twoscomp+1 # add 1
+        hexstr=hex(twoscomp)[2:] #convert to hex but remove the first two 0x characters of the string
+        
+    #if positive or zero just convert to hexidecimal    
+    else: 
+        hexstr=hex(counts)[2:] #convert to hex but remove the first two 0x characters of the string
+        for i in range(8-len(hexstr)):
+            hexstr="0"+hexstr #add zeros to the front of the string until it is 8 characters long
+       
+    print("d2t hxstr: ",hexstr)
+    
+    return hexstr
+
+#performs the reverse operation of deg2tx
+def rx2deg (reply):
+
+    print("r2d reply: ",reply)
+    degPerCount = 0.0025
+    F8=int('FFFFFFFF',16)
+    
+    reply=str(reply)[5:13]#convert reply string from bytes literal and take only the char that correspond to position
+    print("r2d abbrvreply: ",reply)
+    print(reply)
+    angle=999.9 #if there is an error with the reply the angle will be 999. This can be improved by raising an error
+
+    #if positiove or 0 convert to int and then calculate the number of degrees that correspond to the count
+    if (reply[0]=="0"):
+        counts=int(reply,16)
+        angle=counts*degPerCount
+        
+    #if negative reverse the 2s complement syntax then convert to int and degrees
+    if (reply[0]=="F"):
+        repint=int(reply,16)
+        twosuncomp=repint-1
+        counts=(F8-twosuncomp)*-1
+        angle=counts*degPerCount
+
+    angle=round(angle,3)
+    print("r2d: ",angle)
+    return angle
+    
+
+def seiral_move_to(dist):
```

Comment:
should this be "serial" (and everywhere below)?

Reply:
Yes, I guess it was still working because it was consistent. But yeah, I will change it